node/src/service.rs

```diff
@@ -423,3 +420,35 @@ pub async fn start_parachain_node(
 
     Ok((task_manager, client))
 }
+
+/// Start a node with the given parachain `Configuration` and relay chain `Configuration`.
+pub async fn start_parachain_node(
+    parachain_config: Configuration,
+    polkadot_config: Configuration,
+    collator_options: CollatorOptions,
+    para_id: ParaId,
+    hwbench: Option<sc_sysinfo::HwBench>,
+) -> sc_service::error::Result<(TaskManager, Arc<ParachainClient>)> {
+    match polkadot_config.network.network_backend {
+        sc_network::config::NetworkBackendType::Libp2p => {
+            start_node_impl::<sc_network::NetworkWorker<_, _>>(
+                parachain_config,
+                polkadot_config,
+                collator_options,
+                para_id,
+                hwbench,
+            )
+            .await
+        }
+        sc_network::config::NetworkBackendType::Litep2p => {
```

Comment:
It doesn't seem this is needed, though I think it is pretty cool offering the option of exploring `Litep2p`.

The PR bringing this to the sdk mentions that there might be future changes and that its current state is just an initial introduction of this new network backend, at the same time it doesn't seem it would increase massively the maintenance complexity of the template.

Reply:
Thanks @al3mart, I have made changes to remove `litep2p`. I agree that feature is overkill for the purpose of the template.